tools/bin/load_test/connection_spec.json

```diff
@@ -0,0 +1,48 @@
+{
+  "sourceId": "replace_source_id",
```

Comment:
One thought from me here, and I think this is a far out concern - there is the possibility of some drift here as connector spec changes.

I'm not very sure how we would guarantee consistency. Bringing this up in case others have thought of it.

Reply:
Yeah, I don't think there's an easy way to avoid drift, I think if the connection_spec changes shape, the load test will break when submitting API calls. As long as our API errors are clear enough, fixes should be straightforward. I don't think versioning the JSON in the cloud repo will help since breaking changes would occur within the OSS repo/API. 

I'm sure there are ways to write a test that would validate this JSON against the openAPI spec defined in OSS, but I suspect it's more trouble than it's worth right now

Reply:
cool - thanks for the discussion guys.

What about leaving a comment in the file summarising what we just said? That way, future us trying to debug this will connect the dots faster that an error might be due to an unrelated change in either the Airbyte Protocol or the connector's spec.

Reply:
Xiaohan added a comment!